command/host_create.go

```diff
@@ -125,9 +128,14 @@ func (c *createHost) waitForLogs(ctx context.Context, comm client.Communicator,
 			batchEnd := time.Now()
 			status, err := comm.GetDockerStatus(ctx, hostID)
 			if err != nil {
-				return errors.Wrapf(err, "error getting docker status")
+				grip.Info(message.WrapError(err, "problem receiving docker logs in host.create"))
```

Comment:
This will end up going to a file on the host running the agent, so we won't see it. It might be better to not wrap the error (so users don't have stacks in their logs) and instead log it to `logger.Task()` or `logger.Execution()`, depending on what kind of error you expect here.